frontend/public/components/dashboard/project-dashboard/inventory-card.tsx

```diff
@@ -163,6 +164,7 @@ export const InventoryCard = () => {
         <ProjectInventoryItem projectName={projectName} model={RouteModel} />
         <ProjectInventoryItem projectName={projectName} model={ConfigMapModel} />
         {canListSecrets && <ProjectInventoryItem projectName={projectName} model={SecretModel} />}
+        <ProjectInventoryItem projectName={projectName} model={AppliedClusterResourceQuotaModel} />
```

Comment:
I would only add this if we also add `ResourceQuota`, but if we update the quota card, it's probably not needed.